data/mods/joltemons/formats-data.ts

```diff
@@ -0,0 +1,4863 @@
+export const FormatsData: {[k: string]: SpeciesFormatsData} = {
+// Outdated, scroll until you see another "//"
```

Comment:
What do you mean by this? Should these be in formats-data in the first place? Does the pet mod use them?

Reply:
If the mod can only be played with randomized teams, they won't ever show up since they don't have any randomizer moves.

Reply:
the original joltemons uses it, with randbats ig there's no need

without the file will there be any errors though?

Reply:
ok, yoshiblaze clarified what it was doing so i removed it